pkg/argocd/app-controller.go

```diff
@@ -84,23 +89,48 @@ func createApplicationControllerDeployment(clientset *kubernetes.Clientset, name
 							},
 						},
 					},
+					Affinity: &corev1.Affinity{
```

Comment:
I'd suggest we remove the affinity rules here. This is only used during bootstrapping and once Argo CD runs, it will manage itself and apply the config from the gitops repo.